tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,58 @@
+"""
+Automates the test OCS-297:-
+1) Create a Storage Class
+2) Create a PVC
+3) Delete the corresponding storage class
+"""
+import logging
+import pytest
+from ocsci.testlib import tier1, ManageTest
+from tests import helpers
+from ocs_ci.ocs import constants
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Deleting a Storage class
+    """
+    global SECRET, PVC, STORAGE_CLASS
+    log.info("Creating RBD Secret")
+    SECRET = helpers.create_secret(constants.CEPHBLOCKPOOL)
```

Comment:
BTW don't we already have those kind of fixtures which creates secret , pvc or storage_cluss under tests/fixtures.py ?  If they provide the same things please reuse them and not write your own here.